frontend/src/charts/storybook/DisparityBarChart.stories.tsx

```diff
@@ -1,41 +0,0 @@
-import React from "react";
-import { Story, Meta } from "@storybook/react/types-6-0";
```

Comment:
Why are you deleting the DisparityBarChart story?

Reply:
Added back.  I think it was an empty file when I deleted it.